client/components/boards/boardsList.styl

```diff
@@ -1,7 +1,7 @@
 $spaceBetweenTiles = 16px
 
 .board-list
-  margin: $spaceBetweenTiles ($spaceBetweenTiles/-2) 0
+  margin: 0 ($spaceBetweenTiles/2) 0 ($spaceBetweenTiles/2)
```

Comment:
Oh btw, I generally use the shorten form if `margin-top == margin-bottom` and `margin-left == margin-right`, ie:

```styl
  margin: 0 ($spaceBetweenTiles/2)
```